docs/release-notes/8.5.asciidoc

```diff
@@ -1,6 +1,15 @@
 [[release-notes-header-8.5.0]]
 == 8.5
 
+[discrete]
+[[release-notes-8.5.3]]
+=== 8.5.2
+
+[discrete]
+[[bug-fixes-8.5.3]]
+==== Bug fixes and enhancements
+* Fixes a bug that caused {elastic-endpoint} to crash when running on busy Linux systems and when protection features for network events and malicious behavior were enabled.
```

Comment:
@jmikell821 do we have style guidelines for when and/or is acceptable to use?

Reply:
@ferullo I think we normally try to avoid using "and/or" because it sounds overly formal and impersonal. In this case, would only using "or" work? Using "or" would tell users that the crash could happen if only one of the two protection features was enabled.

Reply:
Revised the summary to the following after discussing it with @ferullo over Slack:

```suggestion
* Fixes a bug that caused {elastic-endpoint} to crash when running on busy Linux systems and when the collection of network events or malicious behavior protection were enabled.
```